crates/llm-local/src/lib.rs

```diff
@@ -381,75 +329,10 @@ fn load_tokenizer(tokenizer_file: &Path) -> anyhow::Result<tokenizers::Tokenizer
 }
 
 fn load_model(model_file: &Path) -> anyhow::Result<BertModel> {
-    let buffer = std::fs::read(model_file)
-        .with_context(|| format!("Failed to read model file {}", quoted_path(model_file)))?;
-    let weights = safetensors::SafeTensors::deserialize(&buffer)?;
-    let vb = VarBuilder::from_safetensors(vec![weights], DType::F32, &candle::Device::Cpu);
+    // TODO: Check if there is a safe way to load the model from the file
```

Comment:
The safe method seems to load only a single safe tensor file.

Reply:
See my other comment below.